arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -1573,6 +1563,29 @@
         "location"
       ],
       "x-ms-azure-resource": true
+    },
+    "SubResource": {
+      "description": "The Sub Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
```

Comment:
StorageAccountInfo had name marked as readOnly. Now it will be required to provide name. This looks incorrect to me.